### PR TITLE
feat: add aboutus section

### PR DESCRIPTION
**Pull request by babitarit**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [ ] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by RickDeb2004** [comment]
image
check this once @babitarit

File: src/components/AboutUs/About.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { Heading1 } from '../shared';
+import Content from '../../../config/content/About';
+import {AboutContainer,AboutContent} from './styles';
+import Carousel from './Carousel';
+
+const AboutUs = () => {
+  return (
+    <>
+      <AboutContainer>
+        <Heading1>About Us</Heading1>
```
Comment (SirSimon162) [inline src/components/AboutUs/About.jsx:11]: Create a styled component for this. Add font weight of 700.

File: src/components/AboutUs/styles.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    text-align: center;
+    margin-top: 0px;
+    overflow: hidden;
+    width: 100%;
+    margin-bottom: 50px;
+  `;
+  export const AboutContent = styled(Body1)`
+    margin-top: 20px;
+    margin-bottom: 50px;
+    width: 80%;
```
Comment (SirSimon162) [inline src/components/AboutUs/styles.js:16]: add font size 16px and line height 24px.

File: src/components/AboutUs/Carousel.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+          dots: false,
+          prevArrow: <CustomPrevArrow />,
+          nextArrow: <CustomNextArrow />,
+        },
+      },
+    ],
+  };
+
+  return (
+    <div className='container'>
```
Comment (SirSimon162) [inline src/components/AboutUs/Carousel.jsx:63]: don't use classNames. Replace all the classNames with styled components.

File: src/components/AboutUs/Carousel.jsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+          rel='stylesheet'
+          href='https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0'
+        />
+      </Helmet>
+      <span class='material-symbols-outlined'>arrow_forward_ios</span>
+    </div>
+  );
+}
+
+function Carousel() {
```
Comment (Lucif3r-in) [inline src/components/AboutUs/Carousel.jsx:36]: Rename this as `AboutCarousel`

**Review by Lucif3r-in — CHANGES_REQUESTED** [review]

**Comment by Lucif3r-in** [comment]
@babitarit Resolve merge conflicts

**Pull request merged**